action.yml

```diff
@@ -1,5 +1,9 @@
-name: 'Playwright'
+name: 'Playwright GitHub Action'
```

Comment:
Is it common for the action's display name to include `GitHub Action`? I'm just wondering if `Playwright` would work, and make sense if you're discovering it in the context of the GitHub Action marketplace.

Reply:
I would have liked just Playwright to work, but GitHub complains about not having a unique name while attempting to draft a new marketplace release:
> Playwright - Name must be unique. Cannot match an existing action, user or organization name.

Interestingly though, if I look at existing actions from GitHub, their names start with a verb (interesting because these names are completely different from naming conventions used by users/organizations 🤷 )
* 'Setup Node.js environment' for [actions/setup-node](https://github.com/actions/setup-node/blob/master/action.yml)
* 'Upload a Build Artifact' for [actions/upload-artifact](https://github.com/actions/upload-artifact/blob/master/action.yml)

That could be another route to take, but not something I'm a fan of. Open to other suggestions